core/transactions/transactions_cleanup.cxx

```diff
@@ -265,17 +262,26 @@ transactions_cleanup::create_client_record(const std::string& bucket_name)
 
 const client_record_details
 transactions_cleanup::get_active_clients(const std::string& bucket_name, const std::string& uuid)
+{
+    if (config_.metadata_collection()) {
+        return get_active_clients(config_.metadata_collection().value(), uuid);
+    }
+    return get_active_clients({ bucket_name, "_default", "_default" }, uuid);
```

Comment:
could you use `couchbase::collection::default_name` and `couchbase::scope::default_name` constants here?

Reply:
ooh good point - was being lazy -- will do.